sympy/core/compatibility.py

```diff
@@ -488,6 +488,14 @@ def default_sort_key(item, order=None):
     The order of terms obtained when using these keys is the order that would
     be obtained if those terms were *factors* in a product.
 
+    It is not useful to directly sort expressions based on their complexity
```

Comment:
perhaps, "Although it is useful for quickly putting expressions in canonical order, it does not sort expressions based on their complexity:"

and hopefully the See Also has "ordered" listed.